webpack.config.js

```diff
@@ -62,6 +79,8 @@ module.exports = function(env = {}) {
         bcoin$: `${nodeModsDir}/bcoin/lib/bcoin-browser`,
         bcash$: `${nodeModsDir}/bcash/lib/bcoin-browser`,
         hsd$: `${nodeModsDir}/hsd/lib/hsd-browser`,
+        react: `${nodeModsDir}/react`,
+        '&local': path.resolve(process.env.BPANEL_PREFIX, 'local_plugins'),
```

Comment:
alias for local plugins. `react` alias is needed for webpack to know where to look for local plugins' react dependency.